### DISCUSSION TITLE
improved /chat command

### DISCUSSION THREAD

**Discussion by MrSharfik** [Zero6992/chatGPT-discord-bot#343] [Ideas]
add to the /chat command the ability to select a chat model like /chatGPT, /chatBard, /chatEdge or /chat [message] and after the message the chat model is selected.

**Comment by Zero6992** [comment]
`/chat-model` can switch between the different model

**Reply by neostalker** [reply]
I think they mean, the ability to due something like "/chat [chat-model] ChatGPT4.0 [message] This is my question to chatgpt. I was able to change the chat model within my message without having to use the chat-model command first then use /chat."
